help_to_heat/frontdoor/views.py

```diff
@@ -84,8 +84,8 @@ def unavailable_supplier_redirect(session_id):
 
     if supplier == "Utility Warehouse":
         next_page_name = "application-closed-utility-warehouse"
-    else:
-        next_page_name = "applications-closed"
+    if supplier == "British Gas":
```

Comment:
We'd rather use an `elif` here to keep the logic as one block rather than starting a new if statement

Reply:
Will change this, thanks!